contracts/insured-vesting-v1/InsuredVestingV1.sol

```diff
@@ -61,29 +61,16 @@ contract InsuredVestingV1 is OwnerRole, ProjectRole {
     mapping(address => UserVesting) public userVestings;
 
     // --- Events ---
-    event AllocationSet(address indexed user, uint256 fundingTokenAmount); // TODO(audit) - oldAmount, refundedAmount
+    event AllocationSet(address indexed user, uint256 fundingTokenAmount, uint256 fundingTokenPreviousAmount, uint256 fundingTokenRefundedAmount);
```

Comment:
Perhaps use new/old terminology to be consistent with https://github.com/exciteddao/excited-contracts/blob/main/contracts/vesting-v1/VestingV1.sol#L49?